docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -643,7 +643,6 @@ sysctl -w net.core.wmem_max = 33554432
 ```
 sysctl -w fs.file-max = 9223372036854775807
 ```
-3. The `vmi.Spec.Domain.Resources.Requests.Memory` of a passt VM with no explicit ports should be at least 2048M.
```

Comment:
Please add a warning that a passt VM with no explicit ports has an increased memory overhead.

Reply:
Added.
Also I've noticed there is still mention of a limitation saying binding to low ports is not possible, I've removed it as well as it's no longer the case.